.github/workflows/update-chain-specs.yml

```diff
@@ -24,6 +24,8 @@ jobs:
     steps:
     - uses: actions/checkout@v3
       with:
+        persist-credentials: false
+        fetch-depth: 0
```

Comment:
Why this `fetch-depth: 0`? How does it relate to the problem?

Reply:
Only somewhat related .. It was needed for testing so that triggering a manual workflow run could successfully update the PR that was created by the cron job run.  I've removed it but it can be added back on a temporary branch if we need to do that again (triggering an action manually lets you choose the branch)